src/ucp/core/ucp_ep.c

```diff
@@ -445,16 +445,30 @@ void ucp_ep_config_key_set_err_mode(ucp_ep_config_key_t *key,
                     UCP_ERR_HANDLING_MODE_PEER : UCP_ERR_HANDLING_MODE_NONE;
 }
 
+ucs_status_t
+ucp_ep_config_err_mode_check_mismatch(ucp_ep_h ep,
+                                      ucp_err_handling_mode_t err_mode)
+{
+    if (ucp_ep_config(ep)->key.err_mode != err_mode) {
+        ucs_error("asymmetric endpoint configuration is not supported, error"
+                  " handling level mismatch");
```

Comment:
minor: maybe exact levels are worth adding to this print?

Reply:
done